src/components/Chip/Chip.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Why do we call this Chip? Seems like we should come up with a better name

Reply:
Chip is the exact term for that component. Tags is another close one but many style libraries including materialize calls it Chip.